src/core/inlines.js

```diff
@@ -165,6 +170,7 @@ export function run(conf) {
       "(?:\\[\\[(?:!|\\\\|\\?)?[A-Za-z0-9\\.-]+\\]\\])",
       "(?:\\[\\[\\[(?:!|\\\\|\\?)?[A-Za-z0-9\\.-]+\\]\\]\\])",
       "(?:\\[=[^=]+=\\])", // Inline [= For/link =]
+      "(?<!`)(?:`[^`]+`)", // Inline `code`
```

Comment:
Should we get a way to escape `` ` ``?

Reply:
Thought about that too... I've not run into that in practice in specs, so didn't bother. Another one is "[= link to `thing` in other spec =]". At least this gives us the 90% use case of `true` `false` `null` `void`... etc.

Reply:
We can just fall back to `<code>` if we get one, so maybe okay for now.

Reply:
yeah, exactly.